Sources/StreamChat/Repositories/SyncRepository.swift

```diff
@@ -75,31 +74,23 @@ class SyncRepository {
     /// 3. Refetch channel lists queries, link only what backend returns (the 1st page)
     /// 4. Clean up local message history for channels that are outdated/will get outdated
     /// 5. Run offline actions requests
-    /// 6. Bump the last sync timestamp
     ///
     /// - Parameter completion: A block that will get executed upon completion of the synchronization
     func syncLocalState(completion: @escaping () -> Void) {
```

Comment:
Can't figure out from the PR description why these changes are introduced. Can you please explain in more details @polqf?